README.md

```diff
@@ -79,21 +79,25 @@ Send one of the Chicago Ethereum Meetup organizers your Ethereum addresses (not
 
    This is the contract address you need to approve: `0xaD4a34A7854138d6A8A0254F793AC3510090e9e5`
    
-   <a name="duckduckgo"></a> Use duckduckgo.com to write `10^19` or `10**19` and have the number appear not in scientific notation (like it does in Google). Because DAI has 18 decimal places, this the equivalent of 10 DAI.
-   
    | DAI amount | String to use when picking amount for smart contracts |
    | ------------- | ------------- |
    | 10 DAI | 10000000000000000000 |
    | 3 DAI  | 3000000000000000000 |
    | 1 DAI  | 1000000000000000000 |
+   
+   Note: If you want to manually check the numbers here, you can use duckduckgo.com to write `10^19` or `10**19` and have the number appear not in scientific notation (like it does in Google). Because DAI has 18 decimal places, 10^19 would be the equivalent of 10 DAI.
 
 1. _Optional but a good practice:_ Take a look at [the contracts](./contract-project) to confirm that they do what we say they do.  [Etherscan link for our contract](https://etherscan.io/address/0xad4a34a7854138d6a8a0254f793ac3510090e9e5)
 
 1. Contribute to a project
 
    Use the `contribute` function in the write tab (after approving our contract to spend DAI!). You'll need to type in the name of the project you're contributing to (see [project nicknames](#project_nicknames) above) as well as the amount you want to contribute.
    
-   Use the same [duckduckgo.com method](#duckduckgo) described above for converting the amount of DAI you want to contribute (e.g. 3 DAI) to the proper number (e.g. `3 * 10^18`)
+   | DAI amount | String to use when picking amount for smart contracts |
```

Comment:
"Example values to enter when picking DAI amount for smart contracts"

Reply:
Thanks! FYI once you start typing, you can use this button in the GitHub UI to make your suggestion directly mergeable into this PR 
<img width="39" alt="Screenshot 2020-02-25 20 18 17" src="https://user-images.githubusercontent.com/2731712/75305520-041bbd00-580c-11ea-9969-1d49c8072ffa.png">

<img width="694" alt="Screenshot 2020-02-25 20 18 50" src="https://user-images.githubusercontent.com/2731712/75305538-0bdb6180-580c-11ea-8e74-62f7bc9bebb2.png">

Reply:
@slgraham Curious if you saw this, just as an FYI